docs/conf.py

```diff
@@ -11,12 +11,14 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 import datetime
+import os
+import sys
 
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-# sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('../'))
```

Comment:
I believe this will depend on the CWD, which is probably not what we want. I think we probably want this instead:

```
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
```